spec/01-unit/29-admin_gui/02-admin_gui_template_spec.lua

```diff
@@ -183,7 +185,7 @@ describe("admin_gui template", function()
       conf.prefix = mock_prefix
 
       if not pl_path.exists(usr_interface_path) then
-        assert(pl_path.mkdir(usr_interface_path))
+        os.execute("mkdir -p " .. usr_interface_path)
```

Comment:
nit/non-blocker, but Penlight has a `mkdir -p` equivalent:

https://lunarmodules.github.io/Penlight/libraries/pl.dir.html#makepath

> create a directory path. This will create subdirectories as necessary!